Requirements/ViewModels/ReqIF/RequirementSpecificationMappingDialogViewModel.cs

```diff
@@ -190,17 +230,48 @@ private void InitializeCommands()
             this.BackCommand.Subscribe(_ => this.ExecuteBackCommand());
 
             this.OkCommand = ReactiveCommand.CreateAsyncTask(x => this.ExecuteOkCommand(), RxApp.MainThreadScheduler);
+            
             this.OkCommand.ThrownExceptions.Select(ex => ex).Subscribe(x =>
             {
                 this.ErrorMessage = x.Message;
                 this.IsBusy = false;
             });
 
+            this.SaveMappingCommand = ReactiveCommand.Create();
+            this.SaveMappingCommand.Subscribe(_ => this.SaveMappingCommandExecute());
+
             this.InspectCommand = ReactiveCommand.Create(canExecuteCommand);
             this.InspectCommand.Subscribe(_ => this.ExecuteInspectCommand());
         }
 
         /// <summary>
+        /// Executes the <see cref="SaveMappingCommand"/>
+        /// </summary>
+        private void SaveMappingCommandExecute()
+        {
+            if (string.IsNullOrWhiteSpace(this.importMappingConfiguration.Name))
+            {
+                var saveDialog = new SavedConfigurationDialogViewModel<RequirementsModuleSettings>(
+                    this.pluginSettingsService,
+                    this.importMappingConfiguration,
+                    ConverterExtensions.BuildConverters());
+
+                this.dialogNavigationService.NavigateModal(saveDialog);
+            }
+            else
+            {
+                var settings = this.pluginSettingsService.Read<RequirementsModuleSettings>();
+                
+                var configurationToUpdateIndex = settings.SavedConfigurations.IndexOf(
+                    settings.SavedConfigurations.Single(x => x.Id == this.importMappingConfiguration.Id));
+                
+                settings.SavedConfigurations[configurationToUpdateIndex] = this.importMappingConfiguration;
+
+                this.pluginSettingsService.Write(settings, ConverterExtensions.BuildConverters());
+            }
+        }
+
+        /// <summary>                                                                                                                                        = new SpecificationTypeMapConverter(this.reqIf, this.session, this.iteration);
```

Comment:
This line looks strange in the diff view. Looks like a copy/paste at the wrong place.